sdcm/report_templates/results_performance.html

```diff
@@ -81,8 +81,12 @@ <h2>{{ stat_name }}</h2>
                    percent_abs = best_stat.percent|replace('%', '')|int %}
                     {% if status == 'Regression' and percent_abs > 4 %}
                         <span class="red fbold">-{{ percent }}</span>
+                    {% elif status == 'Regression' and percent_abs < 4 %}
+                        <span class="blue fbold">-{{ percent }}</span>
```

Comment:
blue is not generally considered a negative color, use yellow. Common practice is:
![image](https://user-images.githubusercontent.com/5269241/79069283-b41e8a00-7cd5-11ea-888a-cf0ddc7ee96e.png)